test/e2e/density.go

```diff
@@ -181,6 +191,26 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			additionalNameStr := strconv.Itoa(minionCount) + "-" + string(util.NewUUID())
+			additionalRCName = "my-hostname-density" + additionalNameStr
+			Logf("Schedling additional Pods to measure startup latencies")
+			config = RCConfig{Client: c,
+				Image:         "gcr.io/google_containers/pause:go",
+				Name:          additionalRCName,
+				Namespace:     ns,
+				PollInterval:  itArg.interval,
+				PodStatusFile: fileHndl,
+				Replicas:      minionCount,
+			}
+			latencies, err := RunRC(config)
+			expectNoError(err)
+			sort.Float64s(latencies)
+			perc50 := latencies[len(latencies)/2]
+			perc90 := latencies[(len(latencies)*9)/10]
+			perc99 := latencies[(len(latencies)*99)/100]
+			Logf("Latencies: %v", latencies)
```

Comment:
Since you now have a list of slow pods I suggest calling HighLatencyKubeletOperations and LogPodMetrics (from https://github.com/GoogleCloudPlatform/kubernetes/pull/9523) so we get a dump of which functions took how long on the given node right in the jenkins run